src/SqlFormatter.php

```diff
@@ -1138,9 +131,9 @@ public function format(string $string, bool $highlight = true) : string
             }
 
             // Display comments directly where they appear in the source
-            if ($token[self::TOKEN_TYPE] === self::TOKEN_TYPE_COMMENT ||
-                $token[self::TOKEN_TYPE] === self::TOKEN_TYPE_BLOCK_COMMENT) {
-                if ($token[self::TOKEN_TYPE] === self::TOKEN_TYPE_BLOCK_COMMENT) {
+            if ($token->isOfType(Token::TOKEN_TYPE_COMMENT) ||
+                $token->isOfType(Token::TOKEN_TYPE_BLOCK_COMMENT)) {
```

Comment:
I think I could use variadics or binary here, what's you're preference @goetas ?

```php
$token->isOfType(Token::TOKEN_TYPE_COMMENT, Token::TOKEN_TYPE_BLOCK_COMMENT);
// or (involves renumbering all the types)
$token->isOfType(Token::TOKEN_TYPE_COMMENT | Token::TOKEN_TYPE_BLOCK_COMMENT);
```

Reply:
I would choose the option that requires less work

Reply:
I think I will go with variadics, so that people don't have to know this is implemented with binary numbers behind the scenes